java/lang/Float/Float.hpp

```diff
@@ -494,30 +454,29 @@ namespace Java {
              *
              * @return  a {hash code} value for this object.
              */
-			long hashCode();
+			int hashCode();

```

Comment:
Please make this function override Object.hashCode()

Reply:
Thanks. Fixed.